src/components/item-details/bid-tab/listing-tab-content.tsx

```diff
@@ -28,14 +29,19 @@ export const ListingTabContent = ({ item }: Props) => {
   });
 
   const onSubmit = async ({ price }: TForm) => {
-    dispatch(
-      listItem({
-        price,
-        tokenId: item.tokenId,
-        itemId: item.itemId,
-        ownerAddress: item.owner.address,
-      })
-    );
+    try {
+      await dispatch(
```

Comment:
Please don't add logic related to actions in components. Leverage this logic to thunks and reducers